### DISCUSSION TITLE
Changing the selected language when following the link in the directory tree

### DISCUSSION THREAD

**Discussion by TSLabPRO** [requarks/wiki#3272] [Error / Bug Report]
Hi,

When using the multilingual wiki and Custom Navigation format, the selected language changes to the default language when you click on the Current Directory link in the directory tree. This link is in the format root/home and must be root/en/home

**Host Info (please complete the following information):**
 - OS: Docker Container (Linux)
 - Wiki.js version: 2.5.170
 - Database engine: PostgreSQL 11.10

![2020-12-04_10-16-12](https://user-images.githubusercontent.com/42937450/101133436-c90bad00-3619-11eb-9cf5-5849c367a0a9.png)

**Comment by KarolWoroch** [comment]
Same problem. When I click (in your case: Beginner's Guide) moves me to page in main language, beacuse there isnt a language prefix in url. In case "Current Directory" there are a lang prefix and this is works fine.
